apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -62,6 +62,9 @@ private static DatabaseAdminSettings CreateDatabaseAdminSettings()
             internal QueryOptions QueryOptions { get; }
             internal Priority Priority { get; }
             internal string Tag { get; }
+            internal SpannerConversionOptions ConversionOptions => _lazyConversionOptions.Value;
+
+            private readonly Lazy<SpannerConversionOptions> _lazyConversionOptions;
```

Comment:
Ditto here, I don't think a `Lazy<T>` is called for.

Reply:
Done.